packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -40,6 +40,8 @@ const DateQuestion = require('./dynamic-components/date/question');
 const TextEntryQuestion = require('./dynamic-components/text-entry/question');
 const { documentTypes } = require('@pins/common');
 
+inputMaxCharacters = Math.min(Number(inputMaxCharacters), 32500);
```

Comment:
This was for a codeQL issue, it didn't like the regex being based off an unsanitised env var, this sets a max value of 32500 chars, the number we have as the default in our config